libopenage/CMakeLists.txt

```diff
@@ -164,4 +164,4 @@ if(NOT APPLE)
 	)
 endif()
 
-install(TARGETS libopenage LIBRARY DESTINATION lib)
+install(TARGETS libopenage DESTINATION lib LIBRARY DESTINATION lib)
```

Comment:
One or more groups of properties may be specified in a single call to the TARGETS form of this command. A target may be installed more than once to different locations. Consider hypothetical targets myExe, mySharedLib, and myStaticLib. The code:
```
install(TARGETS myExe mySharedLib myStaticLib
        RUNTIME DESTINATION bin
        LIBRARY DESTINATION lib
        ARCHIVE DESTINATION lib/static)
install(TARGETS mySharedLib DESTINATION /some/full/path)
```
will install myExe to <prefix>/bin and myStaticLib to <prefix>/lib/static. On non-DLL platforms mySharedLib will be installed to <prefix>/lib and /some/full/path. On DLL platforms the mySharedLib DLL will be installed to <prefix>/bin and /some/full/path and its import library will be installed to <prefix>/lib/static and /some/full/path.

---

what i've commited looks like a shorthand version of that example